libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -585,6 +585,7 @@ namespace SohImGui {
                 EnhancementCheckbox("Animated Link in Pause Menu", "gPauseLiveLink");
                 EnhancementCheckbox("Disable LOD", "gDisableLOD");
                 EnhancementCheckbox("Enable 3D Dropped items", "gNewDrops");
+                EnhancementCheckbox("Faster Block Push", "gFasterBlock");
```

Comment:
Given that there are a lot of enhancements. I would suggest gFasterBlockPush.

Variables should be instantly understandable and not vague in anyway (this is just my opinion though, others may disagree).